hastexo/tasks.py

```diff
@@ -364,6 +365,20 @@ def check_stack(self, stack, was_resumed):
                     paramiko.ssh_exception.SSHException,
                     paramiko.ssh_exception.NoValidConnectionsError):
                 self.sleep()
+            except Exception as e:
+                if was_resumed:
+                    error_status = 'RESUME_FAILED'
+                    cleanup = 1
```

Comment:
Any chance we could get descriptive constant names here, instead of the rather opaque `1` and `2`?

Reply:
Fair enough.